assessmentModel/src/iosMain/kotlin/ios.kt

```diff
@@ -49,4 +50,17 @@ actual object DateGenerator {
     }()
 
     actual fun currentYear(): Int = NSCalendar(NSISO8601Calendar).component(NSCalendarUnitYear, NSDate.now).toInt()
+
+    actual fun bridgeIsoDateTimeString(instant: Instant): String {
+        val timeInterval: NSTimeInterval = instant.toEpochMilliseconds() / 1000.0
+        val date = NSDate.dateWithTimeIntervalSince1970(timeInterval)
+        return iso8601Formatter.stringFromDate(date)
+    }
+
+    actual fun instantFromBridgeIsoDateTimeString(dateString: String) : Instant {
+        val date = iso8601Formatter.dateFromString(dateString)
+        val timeInterval = date!!.timeIntervalSince1970()
```

Comment:
Since this is used in serialization which means data that *could* be improperly formatted, I think it would be better to throw an exception (that the serializer can catch) instead of force unwrapping the date.